aspnetcore/mvc/views/razor.md

```diff
@@ -60,6 +60,14 @@ With the exception of the C# `await` keyword, implicit expressions must not cont
 <p>@await DoSomething("hello", "world")</p>
 ```
 
+Implicit expressions can't contain C# generics, as the characters inside the brackets (`<>`) are interpreted as an HTML tag:
```

Comment:
The expression needs one more slight change (@Rick-Anderson might be thinking along these lines, too) ...

> the characters inside the brackets (`<>`) are interpreted as an HTML tag

... makes it sound like "the characters" without the angle brackets are interpreted as the tag.

How about ...

> ... the characters with the brackets (`<>`) is interpreted as an HTML tag

... WRT "is" vs "are" here, "the characters with the brackets (\`<>\`)" is interpreted as a single unit, so the verb is "is" in this case.

Reply:
This change makes sense to me.  I thought that "characters" (plural) was the subject, so it needed "are", but I guess since it's going with "**an** HTML tag" the singular helping verb makes sense.  This is more thinking about grammar than I've done in a while lol

Reply:
True ... and it can be taken *even a step further* ...

> Implicit expressions can't contain C# generics, as the characters inside the brackets (`<>`) are interpreted as an HTML tag:

The first part of the sentence uses plurals, so it's difficult to switch to singular after the comma. An *even better* way to go is to preserve singular everywhere ...

> An implicit expression can't contain a C# generic method, as the characters with the brackets (`<>`) is interpreted as an HTML tag:

Of course, these little nitty grammary things get updated on each successive pass of reviews on topics. They get better and better *and better* as we go. I don't think anyone expects perfection on one pass or one review. It's more like a massaging process to get them into stellar form.

You write well ..... and ur in good company here. My MS is bio. I've noticed after years of teaching that science majors (and English majors, of course) write well, possibly due to the writing demands of the curriculum.

Reply:
Yep I think we've squeezed about the best subject-verb agreement we'll get out of that sentence 😛 I know @Rick-Anderson has some other edits in the works, so until I hear otherwise I'll assume that you or he is making these changes.

> I've noticed after years of teaching that science majors (and English majors, of course) write well

Funny, I've always found the opposite.  Usually, the more technical the person, the worse the writing (big overuse of passive voice was always the one that drove me crazy). Maybe that's more *engineers* than scientists though.

> and ur in good company here. My MS is bio.

Nice, such a rare background in this industry!  I've always loved bio, but for a programmer, spending all day in Excel macros and Matlab scripts gets pretty dry haha.  Glad to know there's another out there!